### PR TITLE
Add D-EDGE

### PR DESCRIPTION
**Pull request by aloisdg**
simpleicons(1)

**Issue:** #6756 
**Alexa rank:** 317,352

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description

- Why did you pick the hex value? The hex value comes from the company branding and the website.
- Did you manually vectorize the logo? no.
- Have you used multiple sources? no.

### CODE REVIEW

File: slugs.md [hunk 1]
```diff
@@ -437,6 +437,7 @@ update the script at 'scripts/release/update-slugs-table.js'.
 | `Cycling '74` | `cycling74` |
 | `Cypress` | `cypress` |
 | `Cytoscape.js` | `cytoscapedotjs` |
+| `D-EDGE` | `dedge` |
```
Comment (dirien) [inline slugs.md:440]: @ericcornelissen, should this be in the PR?

**Review by dirien — APPROVED** [review]
LGTM and a perfect fit.

image

Not sure with the slugs.md So i wait for feedback from @ericcornelissen

File: slugs.md (hunk 1 above)
Comment (ericcornelissen) [reply slugs.md:440]: It's not needed but it's also not a problem

Comment (aloisdg) [reply slugs.md:440]: There is a lot of similar case currently:
```
A-Frame 	aframe
Audio-Technica 	audiotechnica
Co-op 	coop
Conda-Forge 	condaforge
````

Comment (ericcornelissen) [reply slugs.md:440]: The values are fine, it's just that you don't need to include this in your PR as this file is automatically updated when there's a new release. But including won't interfere with that automation so it's also not a problem if it's included

Comment (aloisdg) [reply slugs.md:440]: Alright

Comment (dirien) [reply slugs.md:446]: Cool, then i can merge it :)

**Pull request merged**